tests/Tests.fsproj

```diff
@@ -1,35 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net461</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
     <OutputType>Exe</OutputType>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="../src/Thoth.Json.Net.fsproj" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="Util.fs" />
-    <Compile Include="..\paket-files\tests\thoth-org\Thoth.Json\tests\Types.fs">
-      <Paket>True</Paket>
-      <Link>paket-files/Types.fs</Link>
-    </Compile>
-    <Compile Include="..\paket-files\tests\thoth-org\Thoth.Json\tests\Encoders.fs">
-      <Paket>True</Paket>
-      <Link>paket-files/Encoders.fs</Link>
-    </Compile>
-    <Compile Include="..\paket-files\tests\thoth-org\Thoth.Json\tests\Decoders.fs">
-      <Paket>True</Paket>
-      <Link>paket-files/Decoders.fs</Link>
-    </Compile>
-    <Compile Include="..\paket-files\tests\thoth-org\Thoth.Json\tests\BackAndForth.fs">
-      <Paket>True</Paket>
-      <Link>paket-files/BackAndForth.fs</Link>
-    </Compile>
-    <Compile Include="..\paket-files\tests\thoth-org\Thoth.Json\tests\ExtraCoders.fs">
-      <Paket>True</Paket>
-      <Link>paket-files/ExtraCoders.fs</Link>
-    </Compile>
+    <Compile Include="Types.fs" />
+    <Compile Include="Decoders.fs" />
+    <Compile Include="Encoders.fs" />
+    <Compile Include="ExtraCoders.fs" />
     <Compile Include="Main.fs" />
   </ItemGroup>
-  <Import Project="..\.paket\Paket.Restore.targets" />
+  <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="6.0.7" />
+    <PackageReference Update="Expecto">
+      <Version>9.0.4</Version>
+    </PackageReference>
+    <PackageReference Update="Mono.Cecil">
```

Comment:
❓ Is Cecil needed if we drop the reflection-based features?

Reply:
I don't even know what it is haha, but I'll check. Probably not, though, based on some quick googling it looks like it has to do with inspecting IL code/reflection